test/e2e/bootstrapper_test.go

```diff
@@ -110,3 +143,18 @@ func getSvcState(svcName string) (svc.State, error) {
 	}
 	return status.State, nil
 }
+
+// removeFileIfExists removes the file given by 'path', and will not throw an error if it does not exist
+func removeFileIfExists(t *testing.T, path string) {
+	err := os.Remove(path)
+	if err != nil {
+		assert.Truef(t, os.IsNotExist(err), "could not remove file %s: %s", path, err)
+	}
+}
+
+// isKubeletRunning checks if the kubelet was able to start sucessfully
+func isKubeletRunning(t *testing.T, logPath string) bool {
```

Comment:
Can this be read by any user or just admin or some user who created it?

Reply:
I believe it is administrator only